src/js/helpers/aboutTheDataHelper.js

```diff
@@ -8,14 +8,7 @@ import { apiRequest } from './apiRequest';
 
 export const aboutTheDataQueryString = (params) => {
     if (!Object.keys(params).length) return '';
-    return `?
-    ${params.fiscalYear ? `fiscal_year=${params.fiscalYear}` : ''}
-    ${params.fiscalPeriod ? `&fiscal_period=${params.fiscalPeriod}` : ''}
-    ${params.search ? `&search=${encodeURIComponent(params.search)}` : ''}
-    ${params.page ? `&page=${params.page}` : ''}
-    ${params.limit ? `&limit=${params.limit}` : ''}
-    ${params.order ? `&order=${params.order}` : ''}
-    ${params.sort ? `&sort=${params.sort}` : ''}`;
+    return `?${params.fiscalYear ? `fiscal_year=${params.fiscalYear}` : ''}${params.fiscalPeriod ? `&fiscal_period=${params.fiscalPeriod}` : ''}${params.search ? `&search=${encodeURIComponent(params.search)}` : ''}${params.page ? `&page=${params.page}` : ''}${params.limit ? `&limit=${params.limit}` : ''}${params.order ? `&order=${params.order}` : ''}${params.sort ? `&sort=${params.sort}` : ''}`;
```

Comment:
Whitespace was getting added to the query param when some of these properties were falsy.

Reply:
Thank you

Reply:
We have a package that does this for us called `query-strings`. 
![image](https://user-images.githubusercontent.com/12897813/101695675-f786ef00-3a42-11eb-8c3d-eb7b664f96af.png)

Reply:
```javascript
import {stringify} from 'query-string`

stringify(obj)

// returns objectKey=objectValue&objectKey2=objectValue2
```